### PR TITLE
[json-rpc] expose created address for CreateAccountEvent

### PR DESCRIPTION
**Pull request by xli**
## Motivation

CreateAccountEvent with created address

## Test Plan

json-rpc integration test pass

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/libra/developers.libra.org, and link to your PR here.)

### CODE REVIEW

**Comment by libra-action** [comment]
**This PR may have modified JSON-RPC server code.**

Breaking changes policy:

1. Chaning JSON-RPC method signatures or removing/modifying fields
   in the response in /V1 will not be accepted.
2. Adding fields is ok.
3. Adding a new JSON-RPC method is ok.

Please ensure you have also done the following to update the docs:

1. Update json-rpc/API-CHANGELOG.md.
2. Add/update type/method documentation under json-rpc/docs.

File: json-rpc/types/src/views.rs [hunk 1]
```diff
@@ -192,7 +192,7 @@ pub enum EventDataView {
         time_rotated_seconds: u64,
     },
     #[serde(rename = "createaccount")]
-    CreateAccount {},
+    CreateAccount { created_address: BytesView },
```
Comment (sblackshear) [inline json-rpc/types/src/views.rs:195]: The Move event looks like

```
struct CreateAccountEvent {
  /// Address of the created account
  created: address,
  /// Role of the created account
  role_id: u64
}
```

I think the Rust event got out of sync (my fault) and we need to add `role_id` here + elsewhere

**Review by sblackshear — APPROVED** [review]

File: json-rpc/docs/type_event.md [hunk 2]
```diff
@@ -148,9 +148,11 @@ Event emitted after minted, destination address received the minted coins.
...
 Event emitted when a new account is created
 
-| Name    | Type   | Description                    |
-|---------|--------|--------------------------------|
-| type    | string | Constant string "createaccount"|
+| Name            | Type   | Description                    |
+|-----------------|--------|--------------------------------|
+| type            | string | Constant string "createaccount"|
+| created_address | string | The address created on-chain   |
+| role_id         | u64    | Role id, see [Move resource RoleId](https://github.com/libra/libra/blob/master/language/stdlib/modules/doc/Roles.md#resource-roleid) for more details |
```
Comment (sblackshear) [inline json-rpc/docs/type_event.md:155]: Could additionally link to LIP2 here: https://github.com/libra/lip/blob/master/lips/lip-2.md#move-implementation

**Pull request closed without merging**